prettify_results.py

```diff
@@ -234,24 +234,15 @@ def results_to_map(results_file, stats_files):
 if __name__ == "__main__":
     PARSER = ArgumentParser()
     PARSER.add_argument("-r", "--resultsFile", dest="results_file",\
-        default=False, help="File with results of SourcererCC (results.pairs).")
+        required=True, help="File with results of SourcererCC (results.pairs).")
     PARSER.add_argument("-s", "--statsFiles", dest="stats_files",\
-        default=False, help="File or folder with stats files (*.stats).")
+        required=True, help="File or folder with stats files (*.stats).")
 
     if len(sys.argv) == 1:
```

Comment:
Because of `PARSER.parse_args()` you can safely remove this `if` condition - `PARSER.parse_args()` will raise and exception if not all required arguments provided by itself.